pyggi/base/program.py

```diff
@@ -234,14 +252,16 @@ def dump(self, contents, file_name):
         """
         return self.engines[file_name].dump(contents[file_name])
 
-    def get_modified_contents(self, patch):
+    def get_modified_contents(self, patch, minify=True):
         target_files = self.contents.keys()
         modification_points = copy.deepcopy(self.modification_points)
         new_contents = copy.deepcopy(self.contents)
         for target_file in target_files:
             edits = list(filter(lambda a: a.target[0] == target_file, patch.edit_list))
             for edit in edits:
                 edit.apply(self, new_contents, modification_points)
+            if minify and len(edits) == 0:
```

Comment:
```python
if minify and not edits
````
would to more pythonic 😺 ([reference](https://stackoverflow.com/questions/53513/how-do-i-check-if-a-list-is-empty))